src/sync/scheduler.ts

```diff
@@ -169,7 +170,10 @@ export const getNextTasks = async (subscription: Subscription, targetTasks: Task
 		};
 	}
 
-	const tasks = getTargetTasks(targetTasks);
+	let tasks = getTargetTasks(targetTasks);
```

Comment:
Please also update the `jira-sync-post.ts` for bellow

```
const determineSyncTypeAndTargetTasks = async (syncTypeFromReq: string, subscription: Subscription): Promise<SyncTypeAndTargetTasks> => {

	if (syncTypeFromReq === "full") {
		return { syncType: "full", targetTasks: undefined };
	}

	if (subscription.syncStatus === "FAILED") {
		return { syncType: "full", targetTasks: undefined };
	}

	return { syncType: "partial", targetTasks: ["pull", "branch", "commit", "build", "deployment"] };
};

```

Reply:
Updated